README.md

```diff
@@ -1,6 +1,6 @@
 # Hello Microverse
 
- Hello Microverse. This is our very first project at microverse. In this project we need to show that, we can use git and github smoothly. We would program a html page, which text color need to be changed using css.
+ Good morning. This is our very first project about portfolio design. In this project we need to learn  how to parse a Figma design to create a UI. In the end, we would program our own portfolio using html & css following the figma design provided by microverse. 
```

Comment:
- [ ] I think the description would be better without the 'Good morning' because it is not always in the mornings that a user would be viewing your repo 😅